tests/foreman/ui_airgun/test_hostcollection.py

```diff
@@ -87,6 +115,24 @@ def vm_content_hosts(request, module_loc, module_repos_collection):
     return clients
 
 
+@fixture
+def vm_content_hosts_module_stream(request, module_loc, module_repos_collection_module_stream):
+    clients = []
+    for _ in range(2):
```

Comment:
any reason to not use double `with` statement and yields for automatic post execution procedures and avoid using direct request calls?

Reply:
There was earlier implementation https://github.com/omkarkhatavkar/robottelo/blob/2dbda39e43af4b007336c90efea97fbeb5bfbc31/tests/foreman/ui_airgun/test_hostcollection.py#L78, I took that as my reference.

Reply:
Okay, but even have implementation like this: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui_airgun/test_contenthost.py#L122

Reply:
OK, I can update that.